app/src/main/kotlin/com/simplemobiletools/dialer/services/CallService.kt

```diff
@@ -27,27 +27,21 @@ class CallService : InCallService() {
 
     override fun onCallAdded(call: Call) {
         super.onCallAdded(call)
-        val number = call.getCallerNumber().orEmpty()
-        if (!call.isOutgoing() && !call.isConference() && isNumberBlocked(number.normalizePhoneNumber())) {
-            call.reject(false, null)
-            return
-        } else {
-            CallManager.onCallAdded(call)
-            CallManager.inCallService = this
-            call.registerCallback(callListener)
+        CallManager.onCallAdded(call)
+        CallManager.inCallService = this
+        call.registerCallback(callListener)
 
-            val isScreenLocked = (getSystemService(Context.KEYGUARD_SERVICE) as KeyguardManager).isDeviceLocked
-            if (!powerManager.isInteractive || call.isOutgoing() || isScreenLocked || config.alwaysShowFullscreen) {
-                try {
-                    callNotificationManager.setupNotification(true)
-                    startActivity(CallActivity.getStartIntent(this))
-                } catch (e: ActivityNotFoundException) {
-                    // seems like startActivity can thrown AndroidRuntimeException and ActivityNotFoundException, not yet sure when and why, lets show a notification
-                    callNotificationManager.setupNotification()
-                }
-            } else {
+        val isScreenLocked = (getSystemService(Context.KEYGUARD_SERVICE) as KeyguardManager).isDeviceLocked
+        if (!powerManager.isInteractive || call.isOutgoing() || isScreenLocked || config.alwaysShowFullscreen) {
+            try {
+                callNotificationManager.setupNotification(true)
+                startActivity(CallActivity.getStartIntent(this))
+            } catch (e: ActivityNotFoundException) {
+                // seems like startActivity can thrown AndroidRuntimeException and ActivityNotFoundException, not yet sure when and why, lets show a notification
                 callNotificationManager.setupNotification()
```

Comment:
we arent handling that AndroidRuntimeException?

Reply:
that crash fix comment was added some time ago (I did not), not sure why AndroidRuntimeException is being thrown and not handled.
maybe an oversight?

Reply:
oh right, I see now. Change it to catching every Exception then, not just ActivityNotFoundException